components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -390,12 +390,20 @@ impl Interpolate for BackgroundSize {
 impl Interpolate for RGBA {
     #[inline]
     fn interpolate(&self, other: &RGBA, progress: f64) -> Result<Self, ()> {
```

Comment:
You've already done it. ;-)
https://github.com/servo/servo/blob/9d320d5a34fe9911266940eb1ce96204d345b678/components/style/properties/helpers/animated_properties.mako.rs#L389